styles.css

```
@@ -2,4 +2,50 @@ body {
     background-color: lightseagreen;
 }
 
+footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    justify-content: center;
+    height: 500px;
+}
+
+footer .footer-container {
+    display: flex;
+    flex-direction: column;
+    width: 700px;
+    border: 2px solid red; 
+    justify-content: center;
```

I think I mislead you before, this should actually be

```suggestion
    align-items: center;
```